frame/collective/src/benchmarking.rs

```diff
@@ -34,6 +34,24 @@ fn assert_last_event<T: Config<I>, I: 'static>(generic_event: <T as Config<I>>::
 	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
 }
 
+fn id_to_remark_data<T: Config<I>, I: 'static>(id: u32, length: usize) -> Vec<u8> {
+	let mut remark_data: Vec<u8> = vec![];
+	let mut value = id;
+	let mut counter = 0;
+	while value > u8::MAX as u32 {
+		value -= u8::MAX as u32;
+		remark_data.push(u8::MAX);
+		counter += 1;
+	}
```

Comment:
@ggwpez Sorry for the previous solution, it was a mess now that I look back at it. Hopefully, this is a better solution, if not I will happily close this MR and learn from someone who manages to solve this in a better way.
The idea is I believe really simple here. We are creating the remark data like the following:
1. If `id < 255` we simply push the `id` to the `remark_data` and fill out the rest of the vector with zeros.
2. If not we subtract id with `255` until we get a value that is less than `255`. Until then we are pushing `255` to the `remark_data`. When id becomes less than `255` we push it to the `remark_data` and fill the rest with zeros.

For example(in this example the length is 5): id = 16 => `remark_data = [16, 0, 0, 0, 0]`
Another example(in this example the length is 5): id = 264 => `remark_data = [255, 9, 0, 0, 0]`

Reply:
If you undo these changes where you multiply with 4: `let bytes_in_storage = 4 * (b + size_of::<u32>() as u32);` then you can just do: `id.to_le_bytes().cycle().take().collect()` in this function and be done with it.

Reply:
Clever solution.